packages/orbit-components/src/TextLink/index.d.ts

```diff
@@ -8,9 +8,9 @@ import * as Common from "../common/common";
 
 declare module "@kiwicom/orbit-components/lib/TextLink";
 
-type Type = "primary" | "secondary";
+export type Type = "primary" | "secondary";
```

Comment:
I don't see this being exported from the Flow counterpart, am I missing something? It probably should be, though, feel free to add it.